app/views/admin/loans/tabs/_tabs.slim

```diff
@@ -1,7 +1,8 @@
 ul class="nav nav-tabs nav-tabs-responsive hidden-print" role="tablist"
   = render "admin/loans/tabs/tab", identifier: "details", label: t("loan.details"), tab_class: "active", link_class: "edit-tab"
   = render "admin/loans/tabs/tab", identifier: "questions", label: t("loan.questions"), tab_class: "", link_class: "questions-tab"
-  = render "admin/loans/tabs/tab", identifier: "timeline", label: t("loan.timeline"), tab_class: "", link_class: "timeline-tab"
+  = render "admin/loans/tabs/tab", identifier: "timeline-table", label: "Timeline - Table", tab_class: "", link_class: "timeline-tab"
```

Comment:
These tab labels are not yet translated, but this is an intermediate feature. Do we need to translate these tabs now?